misc/services/mobile-webui/mobile-webui-frontend-testing/tests/spec/picking/picking.spec.js

```diff
@@ -147,3 +147,57 @@ test('Test picking line complete status - draft | in progress | complete', async
 
     await PickingJobScreen.complete();
 });
+
+test.describe('Picking Job Completion', () => {
+
+    test("Should fail when partial picking and allowCompletingPartialPickingJob = N", async ({ page }) => {
+      const { login, pickingSlotQRCode, documentNo, huQRCode, luPIName } =
+        await createMasterdata({
+          allowCompletingPartialPickingJob: false,
+          allowSkippingRejectedReason: true,
```

Comment:
what does `allowSkippingRejectedReason`? why is needed?

since this playwright tests shall be 100% understandable for the QA team,
@cristinastefan90  do you understand it? I not, pls list your questions/clarifications requests?

Reply:
that allows you to partially pick without having to book the rejected qty. I'm taking it out as it seems it creates confusion, the idea was to do at little as possible in the test, as the focus is on the completing of partial pickings, but at the same time, I can see why it is rasing questions as by default that is on false, so I should test with default in mind 👍